mineinabyss-features/src/main/kotlin/com/mineinabyss/displayLocker/DisplayLockerListener.kt

```diff
@@ -1,58 +1,70 @@
-package com.mineinabyss.armorstandlock
+package com.mineinabyss.displayLocker
 
 import com.mineinabyss.components.armorstandlock.LockArmorStand
 import com.mineinabyss.components.armorstandlock.lockedStand
 import com.mineinabyss.components.playerData
+import com.mineinabyss.geary.papermc.access.toGeary
 import com.mineinabyss.geary.papermc.access.toGearyOrNull
 import com.mineinabyss.geary.papermc.store.encodeComponentsTo
+import com.mineinabyss.geary.papermc.systems.WearableItemListener.rightClickArmorStand
+import com.mineinabyss.idofront.messaging.broadcast
 import com.mineinabyss.idofront.messaging.error
+import com.mineinabyss.idofront.messaging.success
 import com.mineinabyss.idofront.spawning.spawn
+import com.mineinabyss.mineinabyss.core.discordSRV
 import org.bukkit.GameMode
 import org.bukkit.Material
 import org.bukkit.Sound
 import org.bukkit.block.Block
 import org.bukkit.block.BlockFace
-import org.bukkit.entity.ArmorStand
-import org.bukkit.entity.Player
-import org.bukkit.entity.Projectile
+import org.bukkit.entity.*
 import org.bukkit.event.EventHandler
 import org.bukkit.event.Listener
 import org.bukkit.event.block.Action
 import org.bukkit.event.block.BlockPistonExtendEvent
 import org.bukkit.event.block.BlockPistonRetractEvent
 import org.bukkit.event.entity.EntityDamageByEntityEvent
+import org.bukkit.event.hanging.HangingPlaceEvent
 import org.bukkit.event.player.PlayerFishEvent
 import org.bukkit.event.player.PlayerInteractAtEntityEvent
 import org.bukkit.event.player.PlayerInteractEvent
 import org.bukkit.inventory.EquipmentSlot
+import org.json.XMLTokener.entity
 
-class ArmorStandLockingListener : Listener {
 
+class DisplayLockerListener: Listener {
     @EventHandler
-    fun PlayerInteractEvent.onPlacingArmorStand() {
+    fun HangingPlaceEvent.onPlaceItemFrame() {
+        val frame = entity as? ItemFrame ?: return
+        val player = player ?: return
+        frame.toGeary().setPersisting(LockArmorStand(player.uniqueId, false, mutableSetOf(player.uniqueId)))
+        player.playerData.recentRightclickedEntity = frame
+        player.error("Use /mia lock toggle to protect this Item Frame")
```

Comment:
perhaps add <b></b> or <st></st> on the commandportion of the string so its obvious what is and isnt? that or italic

Reply:
fixed kinda in [ab91c86](https://github.com/MineInAbyss/MineInAbyss/pull/159/commits/ab91c86505a9eb3a171879287a53a93c13223c87)